Sources/Sentry/include/SentryCrashWrapper.h

```diff
@@ -3,6 +3,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef uint64_t bytes;
```

Comment:
`l`: What about adding a file called `SentryInternalDefines.h` or something to avoid duplication?